stacker_blueprints/dynamodb.py

```diff
@@ -56,3 +94,129 @@ def create_template(self):
                                     Value=GetAtt(table, "StreamArn")))
             t.add_output(Output("{}Name".format(table.title),
                                 Value=Ref(table)))
+
+
+class AutoScaling(Blueprint):
+    """Manages the AutoScaling of DynamoDB tables.
+
+    Ref: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#cfn-dynamodb-table-examples-application-autoscaling # noqa
+
+    Example::
+
+      - name: dynamodb-autoscaling
+        class_path: stacker_blueprints.dynamodb.AutoScaling
+        variables:
+          Tables:
```

Comment:
Regarding our recent discussion, a possible solution for the multiplicity (and avoiding having to use several stacks) would be something like
```
variables:
  AutoScalingGroups:    # or w/e name
    - Tables:
       ...
      ReadCapacity: ...
      WriteCapacity: ...
    - Tables:
     # ....
```
and just `for`-loop it (or a `dict`, however that forces the user to come up with arbitrary names)